controller/work_item_link_blackbox_test.go

```diff
@@ -103,8 +103,10 @@ func (s *workItemLinkSuite) cleanup() {
 	require.Nil(s.T(), db.Error)
 	db = db.Unscoped().Delete(&link.WorkItemLinkCategory{Name: "test-user"})
 	require.Nil(s.T(), db.Error)
-	db = db.Unscoped().Delete(&space.Space{ID: s.userSpaceID})
-	require.Nil(s.T(), db.Error)
+	if s.userSpaceID != uuid.Nil {
```

Comment:
Normally adding a condition in a test is code smell. And the DB utility should handle the delete operation, right?